docs/root/intro/arch_overview/other_features/aggregate_cluster.rst

```diff
@@ -0,0 +1,34 @@
+.. _arch_overview_aggregate_cluster:
```

Comment:
drive by: I think this belongs in the "upstream clusters" arch overview section.

Reply:
Moved document to upstream.